modules/system.yml

```diff
@@ -0,0 +1,19 @@
+- module: system
+  metricsets:
+    - "cpu"
+    - "diskio"
+    - "memory"
+    - "network"
+    - "filesystem"
+    - "fsstat"
+    - "load"
+    - "process"
+    - "process_summary"
+    - "raid"
+    - "socket"
+    - "socket_summary"
+    - "uptime"
+  cpu.metrics: ["percentages", "normalized_percentages"]
+  period: 10s
+  hosts: []
```

Comment:
`hosts` should be empty?

Reply:
It's actually should be removed completly